site/src/Util/Notification.php

```diff
@@ -0,0 +1,177 @@
+<?php
+
+namespace App\Util;
+
+use App\Entity\User;
+use DateTime;
+
+/**
+ * Data class for all Notifications
+ *
+ * @author Aquib Baig <aquibbaig97@gmail.com>
+ */
+class Notification
+{
+    /**
+     * Subject of the Notification
+     *
+     * @var string $subject
+     */
+    protected $subject;
+
+    /**
+     * Actual definition of the Notification
+     *
+     * @var string $message
+     */
+    protected $message;
+
+    /**
+     * User associated with the Notification
```

Comment:
Please change this comment to match the variable (it's the recipient of the notification)